### ISSUE TITLE
How do you install from a repo, rather than the npm registry?

### ISSUE THREAD

**Issue by adambowles** [yarnpkg/yarn#621]

**Comment by kittens** [comment]
Hey @adambowles! Thanks for the question. Can you clarify what you mean? Are you referring to a git repo? If so, you can do this the same as you would with npm. Happy to help and provide commands if you can be more specific. Thanks!

**Comment by farism** [comment]
@kittens It looks like yarn does not support npm supported shorthand syntax e.g. `facebook/react.git`

**Comment by adambowles** [comment]
Hey,

I have this in my `package.json`:
`"package-name": "git+ssh://git@git.mycompany.com:libraries/app-switcher.git"`
(Privately hosted git server)

yarn gives the following error:
```
info No lockfile found.
[1/4] Resolving packages...
error Couldn't find package "package-name" on the "npm" registry.
info Visit http://yarnpkg.com/en/docs/cli/install for documentation about this command.
```

**Comment by montogeek** [comment]
@adambowles Addressed here #513
